RHEL/7/input/system/auditing.xml

```diff
@@ -565,25 +585,33 @@ to the system time should be audited.</rationale>
 
 <Rule id="audit_rules_time_stime">
 <title>Record Attempts to Alter Time Through stime</title>
-<description>On a 32-bit system, add the following to <tt>/etc/audit/audit.rules</tt>:
+<description>If the <tt>auditd</tt> daemon is configured to use the
+<tt>augenrules</tt> program to read audit rules during daemon startup (the
+default), then on a 32-bit system add the following lines to a file with suffix
+<tt>.rules</tt> in the directory <tt>/etc/audit/rules.d</tt>:
 <pre># audit_time_rules
 -a always,exit -F arch=b32 -S stime -k audit_time_rules</pre>
-On a 64-bit system, the "-S stime" is not necessary. The -k option allows for 
-the specification of a key in string form that can be used for better 
-reporting capability through ausearch and aureport. Multiple system calls 
-can be defined on the same line to save space if desired, but is not required.
-See an example of multiple combined syscalls:
-<pre>-a always,exit -F arch=b64 -S adjtimex -S settimeofday -S clock_settime 
--k audit_time_rules</pre>
+On a 64-bit system, the "-S stime" is not necessary. If the <tt>auditd</tt>
+daemon is configured to use the <tt>auditctl</tt> utility to read audit rules
+during daemon startup, then on a 32-bit system add the following lines to
+<tt>/etc/audit/audit.rules</tt> file:
+<pre># audit_time_rules
+-a always,exit -F arch=b32 -S stime -k audit_time_rules</pre>
+On a 64-bit system, the "-S stime" is not necessary. The -k option allows for
+the specification of a key in string form that can be used for better reporting
+capability through ausearch and aureport. Multiple system calls can be defined
+on the same line to save space if desired, but is not required. See an example
+of multiple combined syscalls:
+<pre>-a always,exit -F arch=b64 -S adjtimex -S settimeofday -S clock_settime -k audit_time_rules</pre>
```

Comment:
At first read, this rule says "on a 32bit system add these lines.... but on a 64bit system you don't need the -S stime"

Compare this to audit_rules_time_settimeofday where specific guidance is given. The language in audit_rules_time_settimeofday is more explicit -- could we use that as a template?